.github/workflows/pr-checks.yaml

```diff
@@ -74,7 +74,7 @@ jobs:
       - name: Install dependency packages
         run: |
          sudo apt-get install jq
-         sudo curl -L -o /usr/bin/opa https://openpolicyagent.org/downloads/v0.48.0/opa_linux_amd64_static
+         sudo curl -L -o /usr/bin/opa https://openpolicyagent.org/downloads/v0.55.0/opa_linux_amd64_static
```

Comment:
@MartinBasti one the acceptance criteria mentions that " The same version of OPA as is used in conftest is used also in github workflow". While I updated the opa version to the most recent one, I was unable to confirm that it matches with conftest correctly. How can I do that?

Reply:
```
podman run --rm -ti docker.io/openpolicyagent/conftest:latest@sha256:93c0fccb97538b24ecb8492c5c988f39d27ec7108d4ce99f217677ad50f4271e  --version
Conftest: v0.44.1
OPA: 0.54.0

```